templates/email_base.html

```diff
@@ -1,38 +1,68 @@
-<html style="color: #515665; font-family: open-sans, Helvetica, sans-serif;">
-<head>
-  <meta charset="utf-8">
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
-      rel="stylesheet">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width">
-</head>
-<style>
+{% load static %}
+{% load premailer %}
```

Comment:
The failed test complained about some CSS, I'm guessing that premailer is causing this?

Reply:
it's the `cssutils` dependency :(
https://bitbucket.org/cthedot/cssutils/issues/60/using-units-of-rem-produces-an-invalid